SwiftDeck/README.md

```diff
@@ -0,0 +1,60 @@
+
+# SwiftDeck
+> Swift wrapper around deck.gl
+[![Swift Version](https://img.shields.io/badge/Swift-5.0-F16D39.svg?style=flat)](https://swift.org/)
+
+SwiftDeck provides basic API around deck.gl, allowing for easy integration within an iOS, or macOS application. It uses `WKWebView` internally, which embeds the content of a deck.gl-based application within it.
+
+![Example](https://i.imgur.com/OjM7y4H.png)
+
+## Requirements
+
+- iOS 11.0+/macOS 10.13+
+- Xcode 11
+
+## Installation
+
+#### CocoaPods - Coming Soon!
+You can use [CocoaPods](http://cocoapods.org/) to install `SwiftDeck` by adding it to your `Podfile`:
+
+```ruby
+platform :ios, '11.0'
+use_frameworks!
+
+pod 'SwiftDeck'
+```
+
+#### Manually
+1. Download and drop ```SwiftDeck.framework``` in your project.
+2. Congratulations!
+
+## Usage example
+
+In order to show the map, you'll need a [Mapbox]([https://www.mapbox.com/](https://www.mapbox.com/)) token, which you'd then pass to `MapProvider.mapbox` like so - `MapProvider.mapbox(key: "[YOUR_TOKEN_HERE]")`.
```

Comment:
Nit: 
- "The map" => "mapbox base maps" (See the "Heatmap with Minimaps" example in https://deck.gl/playground which shows two base maps.